projects/burnr/src/utils/constants.ts

```diff
@@ -80,57 +74,18 @@ export const JS_WASM_PROVIDERS: Record<string, LazyProvider> = {
  * These fallback providers connect to a centralized remote RPC node.
  */
 export const REMOTE_PROVIDERS: Record<string, LazyProvider> = {
```

Comment:
Does it make sense for this to be a `Record<T, U>` at all anymore?  There's now only one provider and I think there will only ever be one provider.  If that's true why not go all the way and refactor to account for that?  It's a bit more work but it will tidy things up and make the code more obvious.

Reply:
Absolutely - I was thinking to keep that a bit more in case we decide to roll back but we can always dive back in git history and retrieve what we may need.

Reply:
@raoulmillais I have gone through the refactor you mention, on removing extra things and leaving only one provider (and as you think there will only ever be one provider) then this also means we need some design changes (e.g. replace dropdown with a label, remove autocomplete/search of networks etc etc).

@goldsteinsveta can maybe share an opinion here.

I think though we should keep the functionality of multiple Networks (not local ones) as parachains can a potential pool that could be added here.

Reply:
I would not remove the selector just yet just because we will be adding parachains soon, as I understand. (If Westend's parachain capability is not planned to be maintained in burnr, then sure thing the selector can go)

Reply:
@raoulmillais I kept the structure as is in order to verify if connection with parachains will be eventually need or not. If not then on later PR we can remove the specific structure and reduce Burnr wallet to usage of only 1 provider - which will reduce code and also redesign parts of the app.

Reply:
Thinking out loud... I guess but should we make burnr do transfers on parachain networks?  These demos are kind of a toy examples and that adds more complexity for us to support and for users to understand.  If we have a dedicated separate parachain demo then it probably covers the needs of demoing substrate/connect with parachains and allows us to show off something different than a balance transfer - something more parachain-specific.  I also feel like having a demo per use-case is a neater narrative for showcasing usage.

Reply:
I see you point. This is a healthy and sane approach for all tech demos at paritytech. Thinking out loud...if "switching between networks" is an interesting toy example, let's separate it into its own `projects/network-switch`, if it's not – let it go, I guess

Reply:
I could say it makes sense what you say @raoulmillais .I will proceed with removing dropdown (and most likely keep it in a network-switch place)